Bug Reporting/ARKEmailBugReporter.m

```diff
@@ -25,7 +25,7 @@
 #import "ARKDefaultLogFormatter.h"
 #import "ARKLogStore.h"
 #import "ARKLogMessage.h"
-
+#import "Aardvark.h"
```

Comment:
without it I'm getting an implicit declaration is invalid in C99 error for `ARKLogScreenshot`. Im not super familiar with OBJC_EXTERN aside from it aiding I'm compatibility between objective-c and objective-c++ so open to alternatives aside from extracting the macros into a different file.